File: easy/3. Learning User Input with Natural Numbers/SumNumbers.java
import java.util.Scanner;

class SumNumbers {
	public static void main(String[] arguments) {
		Scanner scanner = new Scanner(System.in);
		int integer = scanner.nextInt();
		int total = 0;

		for (int i = 0; i <= integer; i++) {
			total += i;
		}
		
		System.out.println(total);
	}
}

File: easy/003. Learning User Input with Natural Numbers/sum_numbers.c
#include <stdio.h>

int main()
{
	int integer = 0;
	scanf("%d", &integer);

	int total = 0;

	for (int i = 0; i <= integer; i++) {
		total += i;
	}

	printf("%d", total);

	return 0;
}